packages/table/src/CellDefinitions/IconCell.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '@availity/icon';
+
+export const BuildIcon = ({ name, title }) => <Icon name={name} title={title} />;
+BuildIcon.propTypes = {
+  name: PropTypes.string,
+  title: PropTypes.string,
+};
+
+const IconCell =
+  (iconConfig) =>
+  ({ value }) =>
+    value ? BuildIcon(iconConfig) : null;
```

Comment:
I don't think we need the `BuildIcon` function/component here. The following should suffice unless you had a use case in mind

```
value ? <Icon name={iconConfig.name} title={iconConfig.title} /> : null
```

Reply:
We have a use case.
1) when we want to display an Icon and a header, that doesn't need a `value?` check.
2) when we want to format the title of the icon with some x value from the object or property.
```
  {
                        Header: BuildIcon({ name: 'flag', title: 'Flag for Followup' }),
                        accessor: 'followup',
                        Cell: ({ value }: CellProps<RemitClaimsFollowup>) => {
                            return value ? BuildIcon({ name: 'flag', title: `Assigned To ${value.username}` }) : null;
                        },
                        className: 'fixed-width-icon'
                    },
```

Reply:
I feel like you could do the same thing with the `Icon` component
```
Header: <Icon name="flag" title="Flag for Followup" />
```

Reply:
True, I found a way around it so I didn't have to use the BuildIcon either way.